### PR TITLE
Sage Tooltips: Namespace as [data-js-tooltip]

### PR DESCRIPTION
**Pull request by AndrwM**
## Description
Updates sage tooltip namespace to follow our js binding pattern (`[data-js-tooltip]`).
Update some js file namespacing

### Steps for testing
1. ensure tooltips work with `[data-js-tooltip]`
2. ensure tooltips _don't_ work with bootstrap powered tooltips in super or admin

## Related
https://kajabi.slack.com/archives/CU5JG6N2E/p1592839312000700

### CODE REVIEW

File: lib/sage-frontend/stylesheets/system/patterns/elements/_tooltip.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@
   }
 }
 
-.sage-tooltip-top {
+.sage-tooltip--top {
```
Comment (philschanely) [inline lib/sage-frontend/stylesheets/system/patterns/elements/_tooltip.scss:29]: 😍

**Review by philschanely — APPROVED** [review]

File: lib/sage-frontend/javascript/system/tooltip.js [hunk 2]
```diff
@@ -3,8 +3,10 @@ Sage.tooltip = function() {
   // ==================================================
   // Variables
   // ==================================================
-  var toolTips = Sage.util.nodelistToArray(document.querySelectorAll("[data-tooltip]"));
-  var toolTipClassname = ".sage-tooltip";
+  const DATA_ATTR = "data-js-tooltip";
```
Comment (teenwolfblitzer) [inline lib/sage-frontend/javascript/system/tooltip.js:6]: 🍻

File: lib/sage-frontend/stylesheets/system/patterns/elements/_tooltip.scss (hunk 1 above)
Comment (teenwolfblitzer) [reply lib/sage-frontend/stylesheets/system/patterns/elements/_tooltip.scss:29]: Say that three times fast: "sage tooltip top"

File: lib/sage-frontend/javascript/system/tooltip.js [hunk 3]
```diff
@@ -26,14 +28,14 @@ Sage.tooltip = function() {
 
   // tooltip template
   function buildToolTip(e) {
-    if (!e.target.hasAttribute("data-tooltip")) return;
+    if (!e.target.hasAttribute(DATA_ATTR)) return;
 
-    var pos = e.target.getAttribute("data-position") || "top";
+    var pos = e.target.getAttribute(`${DATA_ATTR}-position`) || "top";
```
Comment (teenwolfblitzer) [inline lib/sage-frontend/javascript/system/tooltip.js:33]: Well, would ya look at that. We can use template literals now! 🥳

**Review by teenwolfblitzer — APPROVED** [review]
👍 🔥 Great work! Thanks for cleaning this one up.

One minor thing: we'll want to update the documentation properties as well, since they still refer to `data-tooltip` rather than the new syntax.

**Pull request merged**